test/integ/cudaOnly/src/cudaNativeFunctions.cpp

```diff
@@ -14,7 +14,7 @@
 #        pragma clang diagnostic push
 #        pragma clang diagnostic ignored "-Wmissing-prototypes"
 #    endif
-__device__ auto userDefinedThreadFence() -> void
+__device__ static auto userDefinedThreadFence() -> void
```

Comment:
just curious - why `static` ?

Reply:
There was the warning, that `userDefinedThreadFence` needs to be defined. I think it means a definition in a header. I used static to change visibility to hidden.

Reply:
GCC throw the exception